<launch>
 <arg name = "robot_laber"/>
 <arg name = "laber_height"/>
 
 <node pkg = "marker" type = "robot_uni_marker.py" name = "robot_uni_marker" output = "screen" respawn = "true" >
  <param name = "robot_laber"     value = "$(arg robot_laber)" />
  <param name = "laber_height"    value = "$(arg laber_height)" />
 </node>
</launch>
